UIKitTemplate/UIKitTemplate/CoffieAndAdditives.swift

```diff
@@ -24,15 +24,33 @@ struct CoffieAndAdditives {
     let aditives = ["Молоко": 50, "Сироп": 20, "Молоко соевое": 50, "Молоко миндальное": 70, "Эспрессо 50мл": 50]
     /// Выбранное кофе
     let coffie: Coffie
+    /// Получения названия выбраного кофе
+    var discriptionCoffie: String {
+        switch coffie {
+        case .americano:
+            return "Американо"
```

Comment:
```suggestion
            return Constants.americano
```
 текстовки в Костанты убираем